src/MPBWrapper.jl

```diff
@@ -2,6 +2,7 @@ module CbcMathProgSolverInterface
 
 using Cbc.CbcCInterface
 using Compat.SparseArrays
```

Comment:
It is very minor but I would find it better to first do `using Compat` before `using Compat.SparseArrays`